tools/llm_bench/task/image_generation.py

```diff
@@ -115,14 +116,29 @@ def run_image_generation_benchmark(model_path, framework, device, args, num_iter
 
     # if num_iters == 0, just output warm-up data
     proc_id = os.getpid()
+    iter_timestamp = {}
     if args['subsequent'] is False:
         for num in range(num_iters + 1):
+            iter_timestamp[num] = {}
             for image_id, image_param in enumerate(image_list):
+                p_idx = prompt_idx_list[image_id]
+                iter_timestamp[num][p_idx] = {}
+                iter_timestamp[num][p_idx]['start'] = datetime.datetime.now().isoformat()
                 run_image_generation(image_param, num, prompt_idx_list[image_id], pipe, args, iter_data_list, proc_id, mem_consumption)
+                iter_timestamp[num][p_idx]['end'] = datetime.datetime.now().isoformat()
+                prefix = '[warm-up]' if num == 0 else '[{}]'.format(num)
+                log.info(f"{prefix}[P{p_idx}] start: {iter_timestamp[num][p_idx]['start']}, end: {iter_timestamp[num][p_idx]['end']}")
     else:
         for image_id, image_param in enumerate(image_list):
+            p_idx = prompt_idx_list[image_id]
+            iter_timestamp[p_idx] = {}
             for num in range(num_iters + 1):
-                run_image_generation(image_param, num, prompt_idx_list[image_id], pipe, args, iter_data_list, proc_id, mem_consumption)
+                iter_timestamp[p_idx][num] = {}
```

Comment:
@wgzintel  Can we just unify the index sequence here?  Without using is_subsequent